README-1.8.md

```diff
@@ -0,0 +1,157 @@
+
+The software in this git repository makes it easy to use SDO-enabled edge devices with Open Horizon. The Horizon SDO support consists of these components:
+
+1. A consolidated docker image of all of the [SDO](https://software.intel.com/en-us/secure-device-onboard) "owner" services (those that run as peers to the Horizon management hub). It also includes a small REST API that enables remote configuration of the SDO OCS owner service.
+1. An `hzn` sub-command to import one or more ownership vouchers into a horizon instance. (An ownership voucher is a file that the device manufacturer gives to the purchaser along with the physical device.)
+1. A sample script called `simulate-mfg.sh` to run the SDO manufacturing components (SCT - Supply Chain Tools) on a test VM device to initialize it with SDO, create the voucher, and extend it to the customer/owner. This script performs the same steps that a real SDO-enabled device manufacturer would.
+1. A script called `owner-boot-device` that initiates the same SDO booting process on a test VM device that runs on a physical SDO-enabled device when it boots.
+
+## Using the SDO Support
+
+Perform the following steps to try out the Horizon SDO support:
+
+- [Start the SDO Owner Services](#start-services) (only has to be done the first time)
+- [Initialize a Device with SDO](#init-device)
+- [Import the Ownership Voucher](#import-voucher)
+- [Boot the Device to Have it Configured](#boot-device)
+
+### <a name="start-services"></a>Start the SDO Owner Services
+
+**Todo: This section can be removed once the SDO owner services are being deployed in the mgmt hub.
+
+The SDO owner services respond to booting devices and enable administrators to import ownership vouchers. These 4 services are provided:
+
+- **RV**: A development version of the rendezvous server, the initial service that every SDO-enabled booting device contacts. The RV redirects the device to the OPS associated with the correct Horizon management hub.
+- **OPS**: The Owner Protocol Service communicates with the devices and securely downloads the device configuration scripts and files.
+- **OCS**: The Owner Companion Service manages the database files that contain the device configuration information.
+- **OCS-API**: A REST API that enables importing and querying ownership vouchers.
+
+The SDO owner services are packaged as a single docker container that can be run on any server that has network access to the Horizon management hub, and that the SDO devices can reach over the network.
+
+1. Get `run-sdo-owner-services.sh`, which is used to start the container:
+
+  ```bash
+  mkdir $HOME/sdo; cd $HOME/sdo
+  curl -sSLO https://raw.githubusercontent.com/open-horizon/SDO-support/stable/docker/run-sdo-owner-services.sh
+  chmod +x run-sdo-owner-services.sh
+  ```
+
+2. Run `./run-sdo-owner-services.sh -h` to see the usage, and set all of the necessary environment variables.
```

Comment:
Wasn't sure if you wanted to include the recent changes for the README in this version or not. I'll add it just in case

2. Run `./run-sdo-owner-services.sh -h` to see the usage, and set all of the necessary environment variables. 

   ```bash
   export HZN_EXCHANGE_URL=https://<cluster-url>/edge-exchange/v1
   export HZN_FSS_CSSURL=https://<cluster-url>/edge-css
   export HZN_ORG_ID=mycluster
   export HZN_EXCHANGE_USER_AUTH=iamapikey:<api-key>
   ```

3. As part of installing the Horizon management hub, you should have run [edgeNodeFiles.sh](https://github.com/open-horizon/anax/blob/master/agent-install/edgeNodeFiles.sh) which created a tar file containing `agent-install.crt`. Use that to export this environment variable:

   ```bash
   export HZN_MGMT_HUB_CERT=$(cat agent-install.crt | base64)
   ```

Reply:
Yes, i did want to. Done